modules.py

```diff
@@ -0,0 +1,71 @@
+import time
+class Product:
+    __id = ''
+    __name = ''
+    __price = 0.0
+    __imageId = ''
+    __description = ''
+    __reviewsList = []
```

Comment:
Move lines 6-8 inside `__init__`. Refer to same link as above for an explanation.